extensions/spine/CCSkeletonCanvasRenderCmd.js

```diff
@@ -132,19 +132,20 @@
     proto._createSprite = function(slot, attachment){
         var rendererObject = attachment.rendererObject;
         var texture = rendererObject.page._texture;
-        var rect = new cc.Rect(rendererObject.x, rendererObject.y, rendererObject.width, rendererObject.height);
         var sprite = new cc.Sprite();
-        sprite.initWithTexture(rendererObject.page._texture, rect, rendererObject.rotate, false);
-        sprite._rect.width = attachment.width;
-        sprite._rect.height = attachment.height;
-        sprite.setContentSize(attachment.width, attachment.height);
-        sprite.setRotation(-attachment.rotation);
-        sprite.setScale(rendererObject.width / rendererObject.originalWidth * attachment.scaleX,
-            rendererObject.height / rendererObject.originalHeight * attachment.scaleY);
-
+        texture.addEventListener('load', function () {
```

Comment:
Texture load event will only be triggered when texture is not yet loaded, if the texture is already loaded, the event won't be triggered